content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Using the TrueNAS CLI Shell"
+description: "This article describes how to use the SCALE CLI Shell for basic networking, updating, and storage management." 
+weight: 130
+---
+
+{{< toc >}}
+
+The TrueNAS CLI Shell functions like a text-based version of the web UI. Users can enter commands to "navigate" to different menus within SCALE and perform actions. This article covers basic operations like setting up networking, performing updates, and listing storage pools/datasets.
+
+## Launch the TrueNAS CLI Shell
+
+To open the TrueNAS CLI Shell, go to the [Console Setup Menu]({{< relref "ConsoleSetupMenuScale.md" >}}) and enter `6`.
+
+![TrueNASCLIlaunch](/images/SCALE/TrueNASCLIlaunch.png "TrueNAS CLI Shell")
+
+To close the TrueNAS CLI Shell, enter `quit`.
+
+### Basic Networking
+
+#### Interfaces
+
+This section covers assigning an IP address to a network interface.
+
+Enter `network interface`.
+
+If you don't already know the interface you want to configure, enter `query` to display a list of all physical network interfaces.
+
+![TrueNASCLInetworkinterfacequery](/images/SCALE/TrueNASCLInetworkinterfacequery.png "Network Interface Query")
+
+To edit the interface, enter `update interfacename aliases=["ipaddress/subnetmask"] ipv4_dhcp=false`
+
+The CLI displays the message: "You have pending network interface changes. Please run 'network interface commit' to apply them."
+
+Enter `commit`, to apply the changes, then enter `checkin` to make them permanant. 
+
+![TrueNASCLIupdateinterfacealiases](/images/SCALE/TrueNASCLIupdateinterfacealiases.png "Update Interface Aliases")
+
+Enter `query` to make sure the Truenas applies the changes successfully.
+
+Enter `..` to exit interface and go up one level to the network menu.
```

Comment:
```suggestion
Enter `..` to exit the interface and go up one level to the network menu.
```

Reply:
```suggestion
Enter `..` to exit `interface` and go up one level to the `network` menu.
```
interface is actually a menu in the CLI. I changed it to be more accurate.